app/editor/src/features/admin/av-overviews/AVOverview.tsx

```diff
@@ -69,90 +76,142 @@ const AVOverview: React.FC = () => {
+              <>
+                <Row className="buttons">
+                  <Select
+                    label="Template Type"
+                    name="templateType"
+                    options={templateTypeOptions}
+                    value={templateTypeOptions.find((o) => o.value === templateType) ?? ''}
+                    onChange={(newValue: any) => {
+                      setTemplateType(newValue.value);
+                    }}
+                    isClearable={false}
+                    width="20ch"
+                  />
+                  <FormikSelect
+                    label="Report Template"
+                    name="templateId"
+                    options={templateOptions}
+                    value={templateOptions.find((o) => o.value === values.templateId) ?? ''}
+                    isClearable={false}
+                  />
+                  <Button
+                    onClick={() => {}}
+                    variant={ButtonVariant.success}
+                    type="submit"
+                    disabled={!values.sections.length}
+                    className="save-items"
+                  >
+                    Save Template
+                    {isSubmitting ? (
+                      <FaSpinner className="icon spinner" />
+                    ) : (
+                      <MdSave className="icon" />
+                    )}
+                  </Button>
+                </Row>
+                {values.sections.map((section, index) => (
+                  <OverviewSection key={index} index={index} />
+                ))}
+                <Row className="buttons">
+                  <Button
+                    variant={ButtonVariant.action}
+                    onClick={() =>
+                      setFieldValue('sections', [
+                        ...values.sections,
+                        defaultAVOverviewTemplateSection(values.templateType),
+                      ])
+                    }
+                  >
+                    New broadcast section <MdAdd className="icon" />
+                  </Button>
+                  <Show visible={!!values.sections.length}>
+                    <Button
+                      onClick={() => {}}
+                      variant={ButtonVariant.success}
+                      type="submit"
+                      className="save-items"
+                    >
+                      Save Template
+                      {isSubmitting ? (
+                        <FaSpinner className="icon spinner" />
+                      ) : (
+                        <MdSave className="icon" />
+                      )}
+                    </Button>
+                  </Show>
+                </Row>
+              </>
+            )}
+          </FormikForm>
+        </Show>
+        <Show visible={active === 'subscribers'}>
+          <Row className="page-header">
+            <div className="title">Update Subscribers</div>
+          </Row>
+          <FormikForm
```

Comment:
Why have a separate form?  Just use one FormikForm that wraps the tabs.

Reply:
Updated, thanks